src/AppBundle/Helper/ApiHelper.php

```diff
@@ -315,4 +317,79 @@ private function massApiInternal(&$data)
             }
         });
     }
+
+    /**
+     * Query the MediaWiki API and return the text of a single page
+     *
+     * @param xtools/Project $project Project that we're working on
```

Comment:
This implies the `Xtools/Project` class, but we pass it into `$this->setUp` which appears to accept a string